ci/courtesy-push/open-courtesy-push-pull-request.js

```diff
@@ -30,6 +30,7 @@ connection.getGitApi().then(gitApi => {
         console.log(`Created PR ${res.url}`);
         const prLink = res.url || 'https://dev.azure.com/mseng/AzureDevOps/_git/AzureDevOps/pullrequests?_a=active&createdBy=fe107a2d-fcce-6506-8e35-5554dbe120fd';
         process.env['PrLink'] = prLink;
+        console.log(`##vso[task.setvariable variable=PrLink]${prLink}`);
```

Comment:
I don't _think_ this will work since setvariable doesn't set the environment variable for the calling step, it only sets it for all subsequent steps. I think it should work if you break this into a separate script though - https://github.com/microsoft/azure-pipelines-tasks/blob/fb0a6b4644b597f2f8879f21692ae3d2471472f8/ci/courtesy-push.yml#L57

Reply:
@damccorm Thank you for the point! Let me do this.